mongodb/README.md

```diff
@@ -0,0 +1,290 @@
+
+```shell
+acorn update [APP-NAME] --arbiter-enabled true
+```
+
+### Removing replicas from an existing replica set
+
+Removing replicas should be done 1 at a time until the desired state is reached. The highest replica indexes will be removed first. If you have 5 replicas running, you can scale down to 3 by updating the app with 1 less than the total running.
+
+```shell
+acorn update [APP-NAME] --replicas 4
+mongosh admin --host $MONGODB_SERVER_LIST --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --eval 'rs.remove("mongodb-5:27017")'
+```
+Before moving on, you should [verify](#how-to-check-replica-set-status) that the replica set is in a good state. If the replica set status is ok, scale down again to 3. If the replica set status is not what you expect, check the logs of all pods to determine if there is an issue.
+
+```shell
+acorn update [APP-NAME] --replicas 3
+mongosh admin --host $MONGODB_SERVER_LIST --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --eval 'rs.remove("mongodb-4:27017")'
+```
+
+Once the desired number of replicas is achieved and the replica set is in a good state, you can also clean up the volumes from the non-existent replicas.
+
+```shell
+> acorn volumes
+NAME                                       APP-NAME   BOUND-VOLUME   CAPACITY   STATUS     ACCESS-MODES   CREATED
+pvc-f457051e-19a4-4d85-8ca0-046b60df060b   mongo    mongodb-data-4   10G        released   RWO            24m ago
+pvc-2b221e93-5476-4368-9622-53cebbbaf2ae   mongo    mongodb-data-2   10G        bound      RWO            24m ago
+pvc-c8a768f7-7328-48d9-ba23-7bf6d7af7148   mongo    mongodb-data-1   10G        bound      RWO            24m ago
+pvc-d5b3bad7-f1de-4eba-ab0d-d671bf4ff84e   mongo    mongodb-data-3   10G        released   RWO            24m ago
+pvc-88f43b37-f201-4478-8e6f-0ed875f2c791   mongo    mongodb-data-0   10G        bound      RWO            84m ago
+
+> acorn rm -v pvc-f457051e-19a4-4d85-8ca0-046b60df060b pvc-d5b3bad7-f1de-4eba-ab0d-d671bf4ff84e
+pvc-f457051e-19a4-4d85-8ca0-046b60df060b
+pvc-d5b3bad7-f1de-4eba-ab0d-d671bf4ff84e
+```
+
+### Trigger election manually
+
+```shell
+
+acorn exec -c mongodb-0 [APP-NAME]
+
+mongosh admin --host $MONGODB_SERVER_LIST --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --eval 'rs.stepDown()'
+
+```
+### Convert standalone to replica set
+
+- Update the acorn app using the following command:
+```shell
+acorn update [APP-NAME] --replicas 1 --is-replica-set true --hidden-replicas 0
+```
+- Initiate the new replica set:
+```shell
+acorn exec -c mongodb-0 [APP-NAME]
+
+mongosh admin --host $MONGODB_SERVER_LIST --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --eval 'rs.initiate()'
+```
+- Check the status of the replica set:
+```shell
+acorn exec -c mongodb-0 [APP-NAME]
+
+mongosh admin --host $MONGODB_SERVER_LIST --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --eval 'rs.status()'
+```
+
+### Backups
+
```

Comment:
Explain here how the backups are taken and stored on a volume in the cluster. In order to survive a catastrophic failure to the cluster. The user should use a storage class that provides volumes from a source outside the cluster.

Reply:
done